specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -412,7 +416,32 @@ function RecordSet<SCHEMA extends AnySchema>({
                 })
             : undefined
         }
-        onClone={(newResource): void => go(totalCount, 'new', newResource)}
+        onClone={(resources) => {
+          resources.map((newResource) => go(totalCount, 'new', newResource));
```

Comment:
doens't make sense to call go multiple times since it will just rapidly jump, changing the URL.
instead, go to the first item of the added resources?

```suggestion
          go(totalCount, 'new', resources[0]);
```

Reply:
agree